src/html/tags/l/Link.php

```diff
@@ -13,7 +13,7 @@ class :link extends :xhp:html-singleton {
   attribute
     enum {'anonymous', 'use-credentials'} crossorigin, Stringish href,
     Stringish hreflang, Stringish media, Stringish rel @required, Stringish sizes,
-    Stringish type;
+    Stringish type, Stringish as;
```

Comment:
The rest of the attributes here are alphabetized, it may be good to preserve that.